apps/web/src/pages/SearchRequests/SearchPage/components/SearchResultCard.tsx

```diff
@@ -14,9 +18,81 @@ const testId = (text: ReactNode) => (
   <span data-testid="candidateCount">{text}</span>
 );
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const SearchResultCard_PoolFragment = graphql(/* GraphQL */ `
+  fragment SearchResultCard_Pool on Pool {
+    id
+    stream {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    publishingGroup {
+      value
+      label {
+        en
+        fr
+      }
+    }
+    classification {
+      group
+      level
+    }
+    name {
+      en
+      fr
+    }
+    department {
+      id
+      name {
+        en
+        fr
+      }
+    }
+    team {
+      id
+      displayName {
+        en
+        fr
+      }
+    }
+    poolSkills {
+      id
+      type {
+        value
+        label {
+          en
+          fr
+        }
+      }
+      skill {
+        id
+        name {
+          en
+          fr
+        }
+        description {
+          en
+          fr
+        }
+        category {
+          value
+          label {
+            en
+            fr
+          }
+        }
+        key
+      }
+    }
+  }
+`);
+
 interface SearchResultCardProps {
   candidateCount: number;
-  pool: Pool;
+  pool: SearchResultCardPoolFragmentType;
```

Comment:
Is there a reason this doesn't just use the fragment type?

```tsx
interface SearchResultCardProps {
  poolQuery?: FragmentType<SearchResultCard_Pool>;
}

const pool = getFragment(SearchResultCard_Pool, poolQuery);
```

Reply:
Nope, no reason at all 
Changed in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11015/commits/774d23086bd459d2a6ecaf1addff4b18b127c257

Reply:
On second thought, that way is a path of terror (fragments are existing in multiple places already and get passed in unpacked here)
Reverted in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11015/commits/b4380d7654d53cbe19b6d4d858e02efd2900b7c1